frontend/public/components/node.tsx

```diff
@@ -78,23 +77,14 @@ const NodeCLUpdateStatus = ({node}) => {
   </div>;
 };
 
-const NodeCLStatusRow = ({node}) => {
-  const updateStatus = containerLinuxUpdateOperator.getUpdateStatus(node);
-  return updateStatus ? <span>{updateStatus.className && <span><i className={updateStatus.className}></i>&nbsp;&nbsp;</span>}{updateStatus.text}</span> : null;
-};
-
 const NodeRow = ({obj: node, expand}) => {
-  const isOperatorInstalled = containerLinuxUpdateOperator.isOperatorInstalled(node);
```

Comment:
We should remove the corresponding `containerLinuxUpdateOperator` code if it's dead code.